sdk/core/Azure.Core/src/Shared/ContentTypeUtilities.cs

```diff
@@ -45,7 +46,8 @@ public static bool TryGetTextEncoding(string contentType, out Encoding encoding)
             if (contentType.StartsWith(textContentTypePrefix, StringComparison.OrdinalIgnoreCase) ||
                 contentType.EndsWith(jsonSuffix, StringComparison.OrdinalIgnoreCase) ||
                 contentType.EndsWith(xmlSuffix, StringComparison.OrdinalIgnoreCase) ||
-                contentType.EndsWith(urlEncodedSuffix, StringComparison.OrdinalIgnoreCase))
+                contentType.EndsWith(urlEncodedSuffix, StringComparison.OrdinalIgnoreCase) ||
```

Comment:
Test to https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/tests/ContentTypeUtilitiesTests.cs#L22

Reply:
Added a couple of OData variants